src/main/java/cat/udl/eps/softarch/demo/repository/PermissionRepository.java

```diff
@@ -0,0 +1,8 @@
+package cat.udl.eps.softarch.demo.repository;
+
+import cat.udl.eps.softarch.demo.domain.Permission;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.PagingAndSortingRepository;
+
+public interface PermissionRepository extends CrudRepository<Permission, String>, PagingAndSortingRepository<Permission, String> {
```

Comment:
If permission has Long as identifier, then the repository should use Long as identity type instead of String